lib/vector-core/src/transform/mod.rs

```diff
@@ -453,10 +463,14 @@ impl OutputBuffer {
         self.0.drain(..).flat_map(EventArray::into_events)
     }
 
-    async fn send(&mut self, output: &mut Fanout) {
+    async fn send(
+        &mut self,
+        output: &mut Fanout,
+    ) -> Result<(), Box<dyn error::Error + Send + Sync>> {
         for array in std::mem::take(&mut self.0) {
-            output.send(array).await;
+            output.send(array).await?;
         }
+        Ok(())
```

Comment:
```suggestion

        Ok(())
```

Same as above, most/all of the newly added `Ok(())`'s get a extra linebreak.